### ISSUE TITLE
error: use of undeclared identifier 'atoi'; did you mean 'stoi'?

### ISSUE THREAD

**Issue by ryandesign** [Edrusb/DAR#30]
**Describe the bug**
dar 2.7.0 fails to build for me on OS X 10.11.6 with the Apple LLVM version 8.0.0 (clang-800.0.42.1) compiler from Xcode 8.2.1:

```
compressor_zstd.cpp:52:20: error: use of undeclared identifier 'atoi'; did you mean 'stoi'?
        U_I min_version = atoi(MIN_MAJ_VERSION_ZSTD)*100*100 + atoi(MIN_MIN_VERSION_ZSTD)*100 + 0;
                          ^~~~
                          stoi
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/string:4105:37: note: 'stoi' declared here
_LIBCPP_FUNC_VIS int                stoi  (const string& __str, size_t* __idx = 0, int __base = 10);
                                    ^
compressor_zstd.cpp:52:57: error: use of undeclared identifier 'atoi'; did you mean 'stoi'?
        U_I min_version = atoi(MIN_MAJ_VERSION_ZSTD)*100*100 + atoi(MIN_MIN_VERSION_ZSTD)*100 + 0;
                                                               ^~~~
                                                               stoi
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/string:4105:37: note: 'stoi' declared here
_LIBCPP_FUNC_VIS int                stoi  (const string& __str, size_t* __idx = 0, int __base = 10);
                                    ^
```

The `-std=c++14` flag has been added manually to CXXFLAGS due to #28.

Apple clang 602.0.49 and later claim to support C++14, so I would expect this to work.

Compiling with newer versions of Apple clang from newer versions of Xcode on newer versions of macOS does work.

I [encountered this error in a different program](https://trac.macports.org/ticket/41855) some time ago and [the fix in that case](https://trac.macports.org/browser/trunk/dports/x11/vnc/files/patch-stdlib.diff?rev=114883) was to add `#include <stdlib.h>`. I have not tried that for dar yet.

**To Reproduce**
Compile dar 2.7.0 with `-std=c++14` in CXXFLAGS at configure time on OS X 10.11.6 with Apple LLVM version 8.0.0 (clang-800.0.42.1)

**Expected behavior**
Successful compile

**Screenshots**
n/a but here is a [full build log](https://build.macports.org/builders/ports-10.11_x86_64-builder/builds/139794/steps/install-port/logs/stdio)

**Software used:**
dar 2.7.0 source code

**Operating system used:**
OS X 10.11.6

**Comment by Edrusb** [comment]
Hi Ryan,

Yes, adding `#include<stdlib.h>`  should fix the issue according to [Mac OS X Man page](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/atoi.3.html)

I will update that shortly

Thanks for feedback,
Denis

**Comment by Edrusb** [comment]
This should be fixed with 2.7.1.RC2 [here](https://dar.edrusb.org/dar.linux.free.fr/Interim_releases/) 
Thanks for your help improving dar/libdar :)

**Comment by Edrusb** [comment]
the just release 2.7.1 should have this fixed now, closing this issue
